modules/videoio/include/opencv2/videoio/legacy/constants_c.h

```diff
@@ -113,6 +113,7 @@ enum
     CV_CAP_PROP_AUTOFOCUS     =39,
     CV_CAP_PROP_SAR_NUM       =40,
     CV_CAP_PROP_SAR_DEN       =41,
+    CV_CAP_PROP_AUTO_WB       =44,
```

Comment:
Please do not touch legacy .h headers. https://github.com/opencv/opencv/blob/4.x/modules/videoio/include/opencv2/videoio.hpp#L181 Is enough for the feature support.

Reply:
I got it. to correct.

Along with this fix, 
existing other Would it be better to change all CV_CAP_PROP enum references in the cap_dshow.cpp backend to videoio.hpp?